vital/types/README.md

```diff
@@ -6,6 +6,8 @@
 
 **camera_map** - Defines the interface base class camera_map and the subclass simple_camera_map, which allow representation of a group of camera instances.
 
+**class_map** - 
```

Comment:
Minor nit: Could you add a description for `class_map` type?

Reply:
Done! Check out 0bfba0a7c388e9555081c4d3bf49228e4b966282. I also changed some of the wordage in the header, as it was specific to the detected_object type.